sdk/ml/azure-ai-ml/.tours/schema_classes.tour

```diff
@@ -0,0 +1,146 @@
+      "line": 15,
+      "title": "Nested Field in YAML"
+    },
+    {
+      "file": "sdk/ml/azure-ai-ml/azure/ai/ml/_schema/compute/schedule.py",
+      "description": "Another important custom field that we make use of is the `UnionField`. This allows us to define polymorphic types. In this example, the \"trigger\" field can be one of two differnt classes through use of more `NestedField` instances.",
+      "line": 82,
+      "title": "UnionField"
+    },
+    {
+      "file": "sdk/ml/azure-ai-ml/azure/ai/ml/_schema/workspace/workspace.py",
+      "description": "Enums also have their own custom field called `StringTransformedEnum`. This field has inputs to let you define the valid enum values, as well as a field to provide a case transformation.",
+      "line": 33,
+      "selection": {
+        "start": {
+          "line": 33,
+          "character": 29
+        },
+        "end": {
+          "line": 33,
+          "character": 50
+        }
+      },
+      "title": "Enums"
+    },
+    {
+      "file": "sdk/ml/azure-ai-ml/azure/ai/ml/_schema/workspace/identity.py",
+      "description": "Schema classes have some annotations that can be used to mark functions for injection at specific points of the YAML conversion process. The `@pre_dump` annotation shown here marks the function below to be run before a python object is converted into a schema class object, and is useful when the two class's fields don't match perfectly.",
+      "line": 43,
+      "title": "@pre_dump annotation"
+    },
+    {
+      "file": "sdk/ml/azure-ai-ml/azure/ai/ml/_schema/workspace/identity.py",
+      "description": "On the other hand, the `@post_load` annotation is needed to define the conversion from a schema class to another (typically more function) class. This function is required for all child schema classes that are defined via `NestedField` values, but it must NOT be defined by any top-level schema class in order to prevent circular import problems.",
+      "line": 49,
+      "title": "@post_load annotation"
+    },
+    {
+      "file": "sdk/ml/azure-ai-ml/azure/ai/ml/entities/_workspace/workspace.py",
+      "description": "The reason for the circular import risk from importing top-level entity classes is the `_load` function that all top-level entity classes are expected to implement, which converts a schema class instance into an entity instance. This is also the top-level intersection point between entity and schema classes.",
+      "line": 148,
+      "title": "Entity Class Relation: _load"
+    },
+    {
+      "file": "sdk/ml/azure-ai-ml/azure/ai/ml/_schema/workspace/workspace.py",
+      "description": "That concludes this introduction to schema classes and how they're used in the AML SDK. If you're curious about them, I encourage you to check out example YAML files that we have strewn about this repo (there are many that are used for testing), and cross-validate them with their schema classes to see how they line up.",
```

Comment:
Is it possible to link to some of the examples in this comment? I think that'd be great.